### PR TITLE
Added gone/log std-lib compatible log library

### PR DESCRIPTION
**Pull request by peter-mogensen**
Please check if what you want to add to `awesome-go` list meets [quality standards](https://github.com/avelino/awesome-go/blob/master/CONTRIBUTING.md#quality-standard) before sending pull request. Thanks!

**Please provide package links to:**
- github.com repo: https://github.com/One-com/gone
- godoc.org: https://godoc.org/github.com/One-com/gone/log
- goreportcard.com: https://goreportcard.com/report/github.com/One-com/gone/log
- coverage service link (gocover, coveralls etc.): http://gocover.io/github.com/One-com/gone/log

**Note**: that new categories can be added only when there are 3 packages or more.

**Make sure that you've checked the boxes below before you submit PR:**
- [x] I have added my package in alphabetical order
- [x] I know that this package was not listed before
- [x] I have added godoc link to the repo and to my pull request
- [x] I have added coverage service link to the repo and to my pull request
- [x] I have added goreportcard link to the repo and to my pull request
- [x] I have read [Contribution guidelines](https://github.com/avelino/awesome-go/blob/master/CONTRIBUTING.md#contribution-guidelines) and [Quality standard](https://github.com/avelino/awesome-go/blob/master/CONTRIBUTING.md#quality-standard).

Thanks for your PR, you're awesome! :+1:

### CODE REVIEW

**Comment by kirillDanshin** [comment]
Please fix link in your commit. Github doesn't support opening repos by import path, so if you want to use a link exactly to your log package, you need to use https://github.com/One-com/gone/tree/master/log. I can't merge a PR with a Not Found response.

**Comment by peter-mogensen** [comment]
Sorry about that ... The link is fixed now.

**Comment by peter-mogensen** [comment]
How did this get the "Pending submitter response" label?
I'm not entirely sure what to respond to. I already have responded to the only issue pointed out.

**Comment by peter-mogensen** [comment]
Could someone elaborate what it is I'm supposed to respond to?
What is the request?

**Comment by peter-mogensen** [comment]
I still have absolutely no idea about what I'm supposed to respond to. :(

**Comment by shurcooL** [comment]
@avelino added that label, he should know.

I suspect it might've been about @kirillDanshin's request to fix link, which you have already done, but perhaps @avelino missed that.

**Comment by peter-mogensen** [comment]
thanks for clearing it up.

File: README.md [hunk 1]
```diff
@@ -516,6 +516,7 @@ See [go-hardware](https://github.com/rakyll/go-hardware) for a comprehensive lis
 * [go-log](https://github.com/ian-kent/go-log) - A log4j implementation in Go.
 * [go-logger](https://github.com/apsdehal/go-logger) - Simple logger of Go Programs, with level handlers.
 * [gologger](https://github.com/sadlil/gologger) - Simple easy to use log lib for go, logs in Colored Cosole, Simple Console, File or Elasticsearch.
+* [gone/log](https://github.com/One-com/gone/tree/master/log) - Fast, extendable, full-featured, std-lib source compatible log library.
```
Comment (shurcooL) [inline README.md]: Optional suggestion, link to beginning of the README in that folder by changing link to:

https://github.com/One-com/gone/tree/master/log#readme

Comment (peter-mogensen) [inline README.md]: good idea... done.

**Comment by shurcooL** [comment]
There are quite a few lint issues, leading to a low score of 36% at https://goreportcard.com/report/github.com/One-com/gone#golint.

You can improve the style of your Go package by addressing those issues.

Generally, packages added to this list have few lint issues and score highly in order to meet the quality standard. Is addressing those issues something you're planning on doing?

**Comment by peter-mogensen** [comment]
I'll see what I can do... many of them, though, are about the deliberate design decision of having level based log functions be all upper case.

**Comment by shurcooL** [comment]
That makes sense. I think it's a valid exception then, since it's done on purpose with some rationale.

**Comment by peter-mogensen** [comment]
Well... I didn't give you the rationale :) ... but the reason is that these very closely mirrors the actual level constants. So if you write l.Log(LOG_ERROR, "...") or l.ERROR("...") it shouldn't change casing. ... also, I personally prefer to have log statements clearly recognizable in the code... and way too much internal code would break if changing it.

**Comment by peter-mogensen** [comment]
score is now 70%

**Comment by kirillDanshin** [comment]
please fix comments as requested in golint standards (it should be of the form "SetFlags ..." not "SetFlags()")
 and I'll merge it. thanks!

**Comment by peter-mogensen** [comment]
there...

**Comment by kirillDanshin** [comment]
awesome!
merged, thanks for your PR :)

**Pull request merged**